apis/stardust/api-spec/spec.json

```diff
@@ -790,13 +790,25 @@
                   "type": "string"
                 },
                 "search": {
-                  "description": "To show the search bar",
+                  "description": "Show the search bar",
                   "optional": true,
                   "defaultValue": true,
                   "type": "boolean"
                 },
                 "toolbar": {
-                  "description": "To show the toolbar",
+                  "description": "Show the toolbar",
+                  "optional": true,
+                  "defaultValue": true,
+                  "type": "boolean"
+                },
+                "checkboxes": {
+                  "description": "Show values as checkboxes instead of as fields",
+                  "optional": true,
+                  "defaultValue": false,
+                  "type": "boolean"
+                },
+                "rangeSelect": {
+                  "description": "Enable range selection",
                   "optional": true,
```

Comment:
I think that there are coming from https://github.com/qlik-oss/nebula.js/pull/764#partial-pull-merging

Reply:
the type definition file should probably also be updated

Reply:
Good point. Thank you. Updated.